hudi-spark/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -52,7 +52,7 @@ private[hudi] object HoodieSparkSqlWriter {
 
     val sparkContext = sqlContext.sparkContext
     val path = parameters.get("path")
-    val tblName = parameters.get(HoodieWriteConfig.TABLE_NAME)
+    val tblName = parameters.get(HoodieWriteConfig.TABLE_NAME).get.trim
```

Comment:
if parameters do not contains HoodieWriteConfig.TABLE_NAME, `parameters.get(HoodieWriteConfig.TABLE_NAME).get.trim` will throw `java.util.NoSuchElementException: None.get`